packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/RefreshButton/UpdateFound/UpdateFound.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect, useRef } from 'react';
+import RefreshIcon from 'react-icons/lib/md/refresh';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+import { UpdateContainer, UpdateMessage } from './elements';
+
+const useInterval = (callback = () => {}, delay: number) => {
```

Comment:
NOTE: This should be replaced with an import from `app/hooks` or an external package. Added here in whole because it's currently part of another PR and this component needs to be tested to see if it works as expected.

Reply:
Resolving as its part of another PR